control-plane/p4RuntimeSerializer.cpp

```diff
@@ -1419,6 +1473,14 @@ class P4RuntimeEntriesConverter {
         return std::string(data.begin(), data.end());
     }
 
+    boost::optional<std::string> stringRepr(cstring value) const {
```

Comment:
what is this for? Which programs require this?

Reply:
New code added to convertSimpleKeyExpression().  But you have said today to now allow Type_Error for table key.  Thus, these code changes will be backed out.